soopercool/metadata_manager.py

```diff
@@ -355,6 +355,22 @@ def get_map_filename(self, map_set, id_split, id_sim=None):
         return os.path.join(path_to_maps,
                             f"{map_set_root}_split_{id_split}.fits")
 
+    def get_filter_kwargs(self):
+        if self.filtering_type == "m_filterer":
+            kwargs = {"m_cut": self.m_cut}
+        elif self.filtering_type == "toast":
+            kwargs = {"schedule": self.toast['schedule'],
+                      "thinfp": self.toast['thinfp'],
+                      "instrument": self.toast['instrument'],
+                      "band": self.toast['band'],
+                      "group_size": self.toast['group_size'],
+                      "nside": self.nside}
+        else:
+            raise NotImplementedError(f"Filterer type {self.filtering_type} "
+                                      "not implemented")
+
+        return kwargs
+
```

Comment:
The filtering functions are defined in utils.py because they contain “physics”. BBmeta is thought of as being “bookkeeping without physics”, so importing the filtering functions from utils.py to BBmeta (or even defining them in-place) would kind of undermine this “rule”…

Reply:
great stuff!